modules/slms_3/1.0/config/default.yaml

```diff
@@ -0,0 +1,180 @@
+lcr-modules:
```

Comment:
It's too bad the skeleton config is needed to allow these values to be updated. I guess the update-config doesn't work if a key isn't already present?

Reply:
The skeleton config is largely to overwrite values that have `__UPDATE__` strings in the module config, and also to highlight the config values that users might want to update.

Reply:
Mostly because the Snakefile is set up to load the default configs from each module.

Reply:
Ah. Makes sense.